Bottom: 9d98240133b546d3969aadbccf36c88f75b093a2
Top:    834726dcc3c814068cf1fbf361f55fbb90dae646
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 13:34:14 -0500

Refresh of exim4-configuration

---
diff --git a/docker/docker-exim4/etc/exim4/update-exim4.conf.conf b/docker/docker-exim4/etc/exim4/update-exim4.conf.conf
index 35f5589a..2e267dd7 100644
--- a/docker/docker-exim4/etc/exim4/update-exim4.conf.conf
+++ b/docker/docker-exim4/etc/exim4/update-exim4.conf.conf
@@ -18,7 +18,7 @@
 
 dc_eximconfig_configtype='internet'
 dc_other_hostnames='novatech-llc.com'
-dc_local_interfaces='127.0.0.1 ; ::1'
+dc_local_interfaces=''
 dc_readhost=''
 dc_relay_domains=''
 dc_minimaldns='false'
